src/docs/self-hosted/logs.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Logs rotation"
+---
+
+By default docker writes logs of a container to one log file which can be giant - tens of gigabytes.
+
+To reduce a used space for logs you can 
+ * modify docker-compose.yaml file by adding into a common section for each container following logging settings:
+
+```
+x-restart-policy: &restart_policy
```

Comment:
This documentation is about log policy, not changing restart policies.
```suggestion
x-log-policy: &log_policy
```

Reply:
My original proposal to solve that problem was with minimal actions because “&restart_policy” is already used by each container.
If we're creating a new common section, we should reformulate following phrase:

    modify docker-compose.yml file and add following common logging setting and apply it either for each container or only for containers with huge log files:

Reply:
It's not clear enough to me this way, also it will affect all containers as maybe users only want to apply this policy to some containers.